.github/workflows/integrate.yaml

```diff
@@ -130,7 +130,7 @@ jobs:
           channel: ${{ matrix.microk8s-versions }}
           microk8s-addons: "dns hostpath-storage rbac metallb:10.64.140.43-10.64.140.49"
           juju-channel: 3.1/stable
-          charmcraft-channel: latest/edge
+          charmcraft-channel: latest/candidate
```

Comment:
Maybe we can link the issue in the comment

Reply:
Resolving as per [this thread](https://github.com/canonical/istio-operators/pull/364#discussion_r1440366926)